foscam.py

```diff
@@ -65,6 +65,10 @@ def send_command(self, cmd, params=None):
         try:
             raw_string = ''
             raw_string = urllib.urlopen(cmdurl).read()
+            if raw:
+                if self.verbose:
+                    print 'Returning raw Foscam response: len=%d' % len(raw_string)
+                return raw_string
```

Comment:
Probably better if we return the same tuple format here for consistency ``(code, data)``. Other than that, this looks great! Thanks.